src/cpp/r/R/Tools.R

```diff
@@ -27,6 +27,9 @@ assign(".rs.toolsEnv", function()
    .rs.Env
 }, envir = .rs.Env)
 
+# Add enviornment with 'tools:rstudio' to store data for Api.R
+assign(".rs.notebookChunkCallbacks", value = new.env(parent = emptyenv()), envir = .rs.toolsEnv())
```

Comment:
Is there a reason why you defined this in `Tools.R` rather than e.g. at the top of `Api.R`? I think you could use `.rs.setVar("notebookChunkCallbacks", new.env(parent = emptyenv())` at the top of `Api.R`.

Reply:
Maybe I'm misunderstanding your point, but the functions `.rs.addFunction()` and `.rs.setVar()`'s main responsibility is to define things in the tools environment. The call you have here is roughly equivalent to:

```
.rs.setVar("notebookChunkCallbacks", new.env(parent = emptyenv()))
```

Reply:
Ah ok, I misunderstood. Fixed 👍